src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -205,7 +205,6 @@ private void UpdateSortIndicator()
 
 		private void FilesystemViewModel_PageTypeUpdated(object? sender, PageTypeUpdatedEventArgs e)
 		{
-			// Show original path and date deleted columns in Recycle Bin
```

Comment:
Removed those comments that are just repeating the few lines below. They are not needed.